lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -153,23 +185,58 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
 
         const size_t instanceIndex = static_cast<size_t>(_instanceIndex);
 
-        PXR_NS::UsdAttribute usdAttr = _getOrCreateAttribute();
-        if (!usdAttr) {
-            return false;
+        // Once setValue() is called, any point instance batch under
+        // construction is closed.  If no batch was created, make a trivial,
+        // unshared batch of a single point instance.
+        _closeBatch();
+
+        PXR_NS::UsdAttribute usdAttr;
+
+        const bool reader = _batch->isReader();
+        const bool writer = _batch->isWriter();
+        _batch->count++;
+
+        if (reader || writer) {
+            usdAttr = _getOrCreateAttribute();
+            if (!usdAttr) {
+                return false;
```

Comment:
Something is really wrong if we enter here with the writer index. Deserves a coding error?

Reply:
Didn't get that one --- what is really wrong?  We need the attribute to read and write.

Reply:
Does it make sense to set values on indexes when there is no attribute? Should we stop it before we get to the last point instance?